src/lib/layout/mobile/NavDrawer.tsx

```diff
@@ -43,37 +48,63 @@ export const NavDrawer = () => {
   const { getSavedPublicProjects } = usePublicProjectStore();
   const { isOpen, onOpen, onClose } = useDisclosure();
 
-  const navMenu = isFullTier
-    ? getNavDrawerFull(
-        govConfig.enabled,
-        wasmConfig.enabled,
-        moveConfig.enabled,
-        nftConfig.enabled
-      )
-    : getNavDrawerLite(
-        govConfig.enabled,
-        wasmConfig.enabled,
-        moveConfig.enabled
-      );
+  const navMenu = useMemo(() => {
+    let navMenuTmp = [];
+    switch (tier) {
+      case "full":
+        navMenuTmp = getNavDrawerFull(
+          govConfig.enabled,
+          wasmConfig.enabled,
+          moveConfig.enabled,
+          nftConfig.enabled
+        );
+        break;
+      case "sequencer":
+        navMenuTmp = getNavDrawerSequencer(
+          govConfig.enabled,
+          wasmConfig.enabled,
+          moveConfig.enabled
```

Comment:
Dont we also need nft enabled?

Reply:
There is no collection list page in sequencer tier.